### DISCUSSION TITLE
Program minimized when Stream Closes

### DISCUSSION THREAD

**Discussion by Zandhoeg** [streamlink/streamlink-twitch-gui#853] [Q&A] [answered]
I have an issue I not noticed before. So not sure if it is new, something I set up differently or something like that.

I usually have GUI and a stream active on a second monitor (GUI minimizing itself when player and stream opens/activates).
On main screen I do gaming or maybe some work stuff etc.

Mainly in full screen games, if the stream ends and the player get closed (as per GUI request and settings) the game minimizes / tabs out to windows and the GUI-software that then get un-minimized so I can select a new stream if I want.

This is something I don't remember being a thing earlier, and I can't say exactly when it started as I usually close a stream before it ends manually. But I am sure I didn't get other active software minimized when streams ended before.

So any tips on what might be the problem, is is GUI, is it the player I use (MPC-HC) or just Windows (Windows 10 21H1) being windows?

**Comment by bastimeyer** [comment, marked as answer]
If you have configured Streamlink Twitch GUI to minimize its window when a stream gets launched, then it'll restore its window again when the stream ends. How this window-restore action is handled depends your operating system / desktop environment and how it implements its window management.

All Streamlink Twitch GUI does is restore the window, nothing related to the window's focus. Window focus does only get requested when authentication, clicking desktop notifications or when clicking the tray bar icon. If your OS steals the current application's focus when another window restores, then there's nothing that can be done here. Since you're using Windows, you unfortunately can't change this behavior.
